source/renderer/app/config/txnsConfig.js

```diff
@@ -1 +1,2 @@
 export const PENDING_TIME_LIMIT = 10 * 60 * 1000; // 10 minutes | unit: milliseconds
+export const TX_DURATION = 2 * 60 * 60; // 2 hours | unit: seconds (7200)
```

Comment:
@tomothespian please rename this variable to: `TIME_TO_LIVE` and change the value of it to: 48 hours.

Reply:
@rvl confirmed this should be set at 2h like you did @tomothespian. Great 🎉